spec/openapi.yaml

```diff
@@ -1138,6 +1160,66 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Job:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 420
+        name:
+          type: string
+          description: Short Filename
+          example: "SPICE~1.gco"
+        display_name:
+          type: string
+          description: Long Filename
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        path:
+          type: string
+          example: "/local"
+        display_path:
+          type: string
+          example: "/PrusaLink gcodes"
+        size:
+          type: integer
+          description: Bytes
+          example: 2393142
+        m_timestamp:
+          type: integer
+          description: Timestamp in seconds
```

Comment:
What kind of timestamp. I think that start job timestamp is fine, but buddy FW have problem with that. What about timestamp of end, or time from end -> and yes, I think, that buddy FW can't supported jobs, which are done.